ddtrace/contrib/kafka/patch.py

```diff
@@ -137,7 +137,7 @@ def traced_produce(func, instance, args, kwargs):
         value = get_argument_value(args, kwargs, 1, "value")
     except ArgumentError:
         value = None
-    message_key = kwargs.get("key", "")
+    message_key = kwargs.get("key", "") or ""
```

Comment:
nit: This fix is very subtle. It would nice to leave a comment here so it's not reverted in a refactor